phpunit.system.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit bootstrap="./tests/system/bootstrap.php" colors="true">
+  <testsuites>
+    <testsuite>
+      <directory>tests/system</directory>
+    </testsuite>
+  </testsuites>
+  <filter>
```

Comment:
This filter isn't needed, unless we're running coverage reports for system tests.